physics.py

```diff
@@ -171,3 +192,31 @@ def update_sim(self, now: float, tm_diff: float) -> None:
             )
             self.vision_sim.update(self.physics_controller.get_pose())
             self.vision_sim_counter = 0
+
+        # Trivially simple wrist simulation
+        self.wrist_motor.setPosition(self.wrist_component.desired_angle)
+        self.wrist_motor.iterate(0.0, 12.0, tm_diff)
+
+        # Simulate algae pick up
+        if self.floor_intake.current_state == "intaking":
+            # Simulate driving around for a couple of seconds
+            self.algae_pickup_counter += 1
+            if self.algae_pickup_counter == 100:
+                self.algae_limit_switch_sim.setValue(False)
+        else:
+            self.algae_pickup_counter = 0
+        if self.reef_intake.current_state == "intaking":
+            # Check near reef
+            pose = self.physics_controller.get_pose()
+            if (pose.translation() - game.BLUE_REEF_POS).norm() < 2.0 or (
+                pose.translation() - game.RED_REEF_POS
+            ).norm() < 2.0:
```

Comment:
nit: use the `.distance()` method
```suggestion
            pose = self.physics_controller.get_pose()
            pos = pose.translation()
            if (
            	pos.distance(game.BLUE_REEF_POS) < 2.0
            	or pos.distance(game.RED_REEF_POS) < 2.0
          	):
```